src/packer.pkr.hcl

```diff
@@ -149,13 +149,180 @@ build {
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    # Install kali packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=kali"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    # Install networking packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=networking"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    # Install password packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=passwords"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    # Install reverse-engineering packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=reverse-engineering"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    # Install utility packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=utilities"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    # Install vulnerability-analysis packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=vulnerability-analysis"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    # Install web-application packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=web-applications"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    # Install word-list packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=word-lists"]
+  }
```

Comment:
Nit-picky but this is how it's always referenced. Would you change the `.config` file name to match as well please?

```suggestion
  provisioner "powershell" {
    # Install wordlist packages
    script           = "src/powershell/install-category.ps1"
    environment_vars = ["CATEGORY=wordlists"]
  }
```